ChangeLog.md

```diff
@@ -1,3 +1,10 @@
+- Adds `Monad` instances for all carrier types.
+- Adds `MonadTrans` instances for almost all carrier types.
+- Adds `Monad` as a superclass of `Carrier`, obviating the need for a lot of constraints.
+- Removes `ret`; `pure` or `return` can be used instead.
```

Comment:
Can we mark the backwards-incompatible changes as such explicitly? We did that in 0.2.0.0 and I think it’s good etiquette to announce “hey, here’s where your code may break."

Reply:
Great call.